cmd/federation.go

```diff
@@ -46,3 +45,39 @@ func main() {
 	s := http.NewServer(config)
 	s.RunServer()
 }
+
+func processActors(logger *log.Log, conn db.Database, q queue.Queue) {
+	actorCQ := queue.ConsumerQueue{
+		QueueName:  "actor_create_queue",
+		Exchange:   "federation",
+		RoutingKey: "actor.create",
+	}
+
+	aw := worker.ActorWorker{
+		Logger:     logger,
+		Repository: repository.NewRepository(conn),
+	}
+
+	err := q.StartConsumer(actorCQ, &aw)
+	if err != nil {
+		logger.Fatal("failed starting actor consumer", err)
+	}
+}
+
+func processPosts(logger *log.Log, conn db.Database, q queue.Queue) {
```

Comment:
This can maybe be made more generic using some of the code from #46

Reply:
Which reminds me... @devanbenz don't delete your branch just yet 😄

Reply:
Sounds good :D